packages/playground/cli/README.md

```diff
@@ -1,76 +1,182 @@
+	"landingPage": "/wp-admin/post-new.php",
+	"steps": [
+		{
+			"step": "installPlugin",
+			"pluginZipFile": {
+				"resource": "wordpress.org/plugins",
+				"slug": "gutenberg"
+			},
+			"options": {
+				"activate": true
+			}
+		},
+		{
+			"step": "login",
+			"username": "admin",
+			"password": "password"
+		}
+	]
+}
+```
+
+The example of a Blueprint above installs a plugin, logs the user in, and opens the new post editor. To learn more about Blueprints, please check the [documentation](https://wordpress.github.io/wordpress-playground/blueprints).
+
+To use a Blueprint, create a file (e.g., my-blueprint.json) and run the following command:
 
-Conceptually, this isn't too different from Docker containers. There are images (zip files),
-containers (running instances), and commands (Blueprints). Playground could support the same
-concepts such as:
+```bash
+npx @wp-playground/cli server --blueprint=./my-blueprint.json
+```
+
+## Programmatic Usage with JavaScript
+
+The Playground CLI can be controlled programmatically from your JavaScript code using the `runCLI` function. This allows you to integrate all CLI functionalities directly into your development workflow, for example, end-to-end testing.
 
--   Listing and managing available images and containers.
--   Saving a running container and restoring it later
--   Starting a container from a specific image (already supported via zip files)
--   Running a command in a container (the `php` command)
--   Building a new image from a Blueprint (the `build` command)
--   Step-by-step cache for Blueprints so that only the changed steps are re-run.
+```JavaScript
+import { runCLI } from "@wp-playground/cli";
+
+const cliServer = await runCLI({
+      command: "server"
+});
+const handler = cliServer.requestHandler;
+const php = await handler.getPrimaryPhp();
+```
 
 ## Interoperability
 
-This CLI package is not just a useful tool. It drives interoperability between the in-browser
-Playground, CLI packages, and the PHP Blueprints library. Once complete, it will reuse the
-same internals as the website at https://playground.wordpress.org whether we're talking about
-running PHP code, executing Blueprints, building snapshots, serving requests, or maintaining
-multiple PHP instances
+This CLI package is not just a helpful tool. It drives interoperability between the in-browser Playground, CLI packages, and the PHP Blueprints library. Once complete, it will reuse the same internals as the website at https://playground.wordpress.net, whether we're talking about running PHP code, executing Blueprints, building snapshots, serving requests, or maintaining
+multiple PHP instances.
+
+## Comparisons
+
+### Things the Playground does compared to Laravel Valet
+
+-   Handles the entire WordPress installation for you.
+-   Works across all desktop platforms (Mac, Linux, Windows).
+-   Does not set up custom host domains for you.
+-   Allows you to switch WordPress and PHP versions on the fly.
```

Comment:
Not quite on the fly, you still need to stop the server and run another command.